src/app/_components/OutlinedBtn/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './OutlinedBtn';
```

Comment:
🟠 what is the point for component of 20 lines long to create folder and 2 files?

Reply:
Deleted.